.circleci/config.yml

```diff
@@ -69,3 +69,58 @@ jobs:
       - store_artifacts:
           path: test-results
           destination: tr1
+
+
+  deploy_pypi:
+    docker:
+      - image: circleci/python:3.7
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      - run:
+          name: configure pypirc
+          command: |
+            echo -e "[pypi]" >> ~/.pypirc
+            echo -e "username = $PYPI_USERNAME" >> ~/.pypirc
+            echo -e "password = $PYPI_PASSWORD" >> ~/.pypirc
+
+      - run:
+          name: setup_venv
+          command: |
+            python3 -m venv venv
+            . venv/bin/activate
+            pip install -U twine setuptools wheel
+
+      - run:
+          name: check_version
+          command: |
+            . venv/bin/activate
+            python setup.py check_version -e $CIRCLE_TAG
+
+      - run:
+          name: build wheels
```

Comment:
Please add notes to the release instructions on how to deploy to pypi with the aid of this config

Reply:
added a note into RELEASE_INSTRUCTIONS.md.